modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/blogPosting/blogPostingJsonT.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
is jsonT a format in json file? Or a file with .jsont suffix?

Reply:
it should have .json extension